### ISSUE TITLE
Simulator's navbar and draggable panels position adjustment (Responsive View)

### ISSUE THREAD

**Issue by bhutani2002** [CircuitVerse/CircuitVerse#2931]
**Is your feature request related to a problem? Please describe.**
The simulator's navbar is just hiding below all the panels when, it's opened . The panels doesn't seem to look nice in the position where they are right now, as on any small screen, the gap between the panels makes uncomfortable for the user to put the elements in between that gap and it's is just like having unnecessary space and degrading frontend look, of the simulator on small screens.

**Describe the solution you'd like**
To make the navbar, placed over the panels when opened. Adjusting the initial positions of the panels, to make them more viewable and for the frontend to not look uneasy for a user.

**Additional context**

**Navbar**
Before 
![image](https://user-images.githubusercontent.com/84590758/153930854-01ab573e-b896-4e22-9b5d-687abf530ff8.png)
After
![image](https://user-images.githubusercontent.com/84590758/153931471-c734a1e6-063a-45cd-be84-a0ec9f941407.png)

**Panels Positioning and view**
Before
![image](https://user-images.githubusercontent.com/84590758/153931866-f6c71d41-1a31-4bfb-a99e-9a24575e63b9.png)
After

Initially just the circuit elements panel will remain open and properties panel will remain closed.

![image](https://user-images.githubusercontent.com/84590758/153932134-8a958ddf-a031-4261-acee-7e444fd87a49.png)

Before
![image](https://user-images.githubusercontent.com/84590758/153932493-3a54db7f-ad25-4745-ab61-5a37e19eb578.png)
After
![image](https://user-images.githubusercontent.com/84590758/153932919-aa2a21d0-be3e-4018-a4da-5035a5b42d0d.png)
![image](https://user-images.githubusercontent.com/84590758/153933058-6be9f630-13ef-4fc0-bb5a-dd0377f9018d.png)
![image](https://user-images.githubusercontent.com/84590758/153933320-4b9aa6c4-9495-4162-911d-8a29fccb6aaf.png)

So, these are few of the screenshots depicting the before and after results.
Please Excuse me for bigger dimensions of the pics. I have not reduced the dimensions. Because if I would have reduced it, the clear differences might not be seen.

**Are you working on this? (Yes/No)**
Yes

**Comment by bhutani2002** [comment]
Hi @gr455 @tachyons @vedant-jain03 , please try to review it once and provide me a feedback so that I can complete the task and generate a pull request.

**Comment by gr455** [comment]
Actually, at the moment, simulator is not compatible with phones and it's not supposed to be used on a phone. We will be making it mobile compatible with new updates

**Comment by bhutani2002** [comment]
> Actually, at the moment, simulator is not compatible with phones and it's not supposed to be used on a phone. We will be making it mobile compatible with new updates

Yes, I read the project list of GSOC'22 and there was something provided to improve on the UI . Therefore I suggested this idea, which I thought might be beneficial. As there's gonna be work, to be done on the touch for mobile devices, here I provided the idea and implementation of the initial positions of the panels. So I just pointed out the positions across various widths that we can have, so that the space is evenly distributed and the user can comfortably use the simulator with proper view. And the navbar is also not properly visible for the small screens. So thought of correcting that too. 

So, if you allow me I can work on that, i.e. on the responsive UI design of simulator as shown above.

Thank You
